src/main/java/org/apache/commons/lang3/math/Fraction.java

```diff
@@ -281,7 +277,6 @@ public static Fraction getFraction(double value) {
             denom2 = a1 * denom1 + denom0;
             fraction = (double) numer2 / (double) denom2;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The result of `Integer` multiplication is cast into a larger type. If you think the result of the multiplication could exceed the size of an `Integer`, consider casting the arguments of the multiplication to avoid arithmetic overflows. For more information, see [CWE-190](https://cwe.mitre.org/data/definitions/190.html).